pkgs/applications/science/logic/anylogic/default.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  gnutar,
+  steam-run,
+  makeDesktopItem,
+  copyDesktopItems,
+}: let
+  desktopItem = makeDesktopItem {
+    desktopName = "AnyLogic Personal Learning Edition";
+    genericName = "AnyLogic Personal Learning Edition - for beginners and students";
+    categories = ["Science"];
+    exec = "anylogic";
+    name = "anylogic-ple";
+    icon = "anylogic-ple";
+  };
+in
+  stdenv.mkDerivation rec {
+    pname = "anylogic-ple";
+    version = "8.8.2";
+
+    src = fetchurl {
+      url = "https://files.anylogic.com/${pname}-${version}.linux.x86_64.tgz.bin";
+      sha256 = "0n372fq6q1pr0y3qxwdf5xxx9phmlyrjjgji861qilrqdg4cmzcp";
+    };
+
+    nativeBuildInputs = [copyDesktopItems gnutar];
```

```suggestion
    nativeBuildInputs = [ copyDesktopItems ];
```
gnutar is part of stdenv IIRC